content/docs/user-guide/index.md

```diff
@@ -1,11 +1,23 @@
 # User Guide
 
 Our guides describe the major concepts in MLEM and how it works comprehensively,
-explaining when and how to use what, as well as inter-relationship between them.
+explaining when and how to use its features.
 
-The topics here range from more foundational (impacting many parts of MLEM) to
-more specific and advanced things you can do. We also include a few misc.
-guides, for example related to [contributing to MLEM](/doc/contributing) itself.
+## Codification: the MLEM way
 
-Please choose from the navigation sidebar to the left, or click the `Next`
-button below ↘
+Saving machine learning models to files or loading them back into Python objects
+may seem like a simple task at first. For example, the `pickle` and `torch`
```

Comment:
> Moved all this to the User Guide

... here 🙂

Reply:
what exactly do you suggest moving?

Reply:
@aguschin here I was just explaining what I did (moving the _codification_ background info to the UG).

If you referred to "Moved all this to the User Guide

... here (edited a little 🙂)

p.s. we can expand on this page, probably summarizing the usage basics and removing some details from the sub-pages (separate task)", to start with, the info from [Basic concepts](https://mlem-ai-start-edits-ojgyvycqih.herokuapp.com/doc/user-guide/basic-concepts) and [Working with models](https://mlem-ai-start-edits-ojgyvycqih.herokuapp.com/doc/user-guide/models) could be put in the UG index (ideally rewritten in a way that explains why should you learn this paradigm, similar to [what we're doing in DVC](https://github.com/iterative/dvc.org/pull/4042) -- I can help).

Reply:
I suggest to move to basic-concepts (see [here](https://github.com/iterative/mlem.ai/pull/281/files#diff-9e5ea1c7881a8d2248846bcfbd5ed700fb26749b25a3fc0fcb335e7c82489c17R29))

Reply:
Hmm. I don't love that page in general (Basic Concepts). What's its purpose? It only covers MLEM Object which is cornerstone so it can probably also be absorbed into the guide index page (or somewhere else).

Also, I rewrote most of the text here @omesser , so it's not the same version as what you have in https://github.com/iterative/mlem.ai/pull/281/files#diff-9e5ea1c7881a8d2248846bcfbd5ed700fb26749b25a3fc0fcb335e7c82489c17